src/main/java/com/google/firebase/auth/FirebaseAuth.java

```diff
@@ -213,12 +214,7 @@ FirebaseTokenVerifier getSessionCookieVerifier(boolean checkRevoked) {
   }
 
   @Override
-  protected void doDestroy() {
-    // Only destroy the tenant manager if it has been created.
-    if (tenantManagerCreated.get()) {
-      getTenantManager().destroy();
-    }
-  }
+  protected void doDestroy() { }
```

Comment:
Why don't we need to need to check if the app has been destroyed in the tenant manager?

Reply:
I think this is something we ought to get rid of from the entire codebase. `FirebaseAuth` is the only class that does this today. I think it's fair to not do it in any new classes.

Reply:
Ack.